tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/stratifiedRules/bool/OrStratifiedRule.scala

```diff
@@ -0,0 +1,76 @@
+package at.forsyte.apalache.tla.bmcmt.stratifiedRules.bool
+
+import at.forsyte.apalache.tla.bmcmt._
+import at.forsyte.apalache.tla.bmcmt.rewriter.ConstSimplifierForSmt
+import at.forsyte.apalache.tla.bmcmt.stratifiedRules.{addCell, StratifiedRule}
+import at.forsyte.apalache.tla.bmcmt.stratifiedRules.{Rewriter, RewriterScope}
+import at.forsyte.apalache.tla.lir.{BoolT1, OperEx, TlaEx}
+import at.forsyte.apalache.tla.lir.oper.TlaBoolOper
+import at.forsyte.apalache.tla.typecomp.TBuilderInstruction
+import at.forsyte.apalache.tla.types.tla
+
+/**
+ * For state-level expressions, we express A \/ B as IF A THEN TRUE ELSE B. For action-level expressions, i.e.,
+ * involving primes, we do a direct translation to A \/ B. This mimics the behavior of TLC.
```

Comment:
I believe this comment is also quite outdated. Is there any code left that distinguishes between action and non-action expressions?